Brokerages/Zerodha/ZerodhaSymbolMapper.cs

```diff
@@ -257,24 +276,62 @@ public Symbol GetLeanSymbol(string brokerageSymbol)
 
 
             var symbol = KnownSymbols.FirstOrDefault(s => s.Value == brokerageSymbol);
-            return GetLeanSymbol(brokerageSymbol, symbol.SecurityType, symbol.ID.Market);
+            var exchange = GetZerodhaDefaultExchange(brokerageSymbol);
+            return GetLeanSymbol(brokerageSymbol, symbol.SecurityType, exchange);
         }
 
         /// <summary>
-        /// Converts an Zerodha symbol to a Zerodha Instrument Token instance
+        /// Fetches the trading segment inside India Market, E.g: NSE, BSE for the given Instrument Token
+        /// </summary>
+        /// <param name="Token">The Zerodha Instrument Token</param>
+        /// <returns>An exchange value for the given token</returns>
+        public string GetZerodhaExchangeFromToken(uint Token)
+        {   
+            string exchange = string.Empty;
+            if (ZerodhaInstrumentsExchangeMapping.ContainsKey(Token))
+            {
+                ZerodhaInstrumentsExchangeMapping.TryGetValue(Token, out exchange);
+            }
+            return exchange;
+        }
+        
+
+        /// <summary>
+        /// Fetches the first available Exchage value for the given symbol from list of possible exchanges
         /// </summary>
         /// <param name="brokerageSymbol">The Zerodha symbol</param>
-        /// <param name="market">The trading market</param>
-        /// <returns>A new Lean Symbol instance</returns>
-        public uint GetZerodhaInstrumentToken(string brokerageSymbol, string market)
+        /// <returns>A default exchange value for the given ticker</returns>
+        public string GetZerodhaDefaultExchange(string brokerageSymbol)
+        {   
+            brokerageSymbol = brokerageSymbol.Replace(" ", "").Trim();
+
+            List<SymbolData> tempSymbolDataList;
+            if (ZerodhaInstrumentsList.TryGetValue(brokerageSymbol.Replace(" ", "").Trim(), out tempSymbolDataList))
```

Comment:
Minor comments:
- `.Replace(" ", "")` is already done above
- All methods/properties that can be private should be
- Commented code should be removed
- All unused methods should be removed, I see `IsKnownLeanSymbol` not being used
- We should generally replace `""` for `string.Empty`
- Do `ZerodhaInstrumentsExchangeMapping/ZerodhaInstrumentsList/KnownSymbolsList` need to be static? Think would avoid them being static because they still depend on an instance initialization